### PR TITLE
Added Admin Course CRUD

### PR DESCRIPTION
**Pull request by codegracie**
-Admin Course for Admin
-Subjects

### CODE REVIEW

**Comment by kian1213** [comment]
Okay

File: app/controllers/admin/courses_controller.rb [hunk 1]
```diff
@@ -0,0 +1,48 @@
+class Admin::CoursesController < AdminController
+  def index
+    if params[:search]
+      @courses = Course.paginate(page: params[:page])
```
Comment (dieunb) [inline app/controllers/admin/courses_controller.rb]: paginate after search

File: app/controllers/admin/courses_controller.rb [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+    @course = Course.find params[:id]
+  end
+
+  def new
+    @course = Course.new
+  end
+
+  def create
+    @course = Course.new course_params
+      if @course.save
```
Comment (dieunb) [inline app/controllers/admin/courses_controller.rb]: indent

File: app/controllers/admin/courses_controller.rb [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+
+  def edit
+  end
+
+  def update
+  end
+
+  def destroy
+   @course = Course.find params[:id]
+      @course.destroy
```
Comment (dieunb) [inline app/controllers/admin/courses_controller.rb]: indent

File: app/controllers/admin/courses_controller.rb [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+      redirect_to admin_courses_path
```
Comment (dieunb) [inline app/controllers/admin/courses_controller.rb]: indent

File: app/controllers/admin/courses_controller.rb [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+  end
+
+  private
+    def course_params
```
Comment (dieunb) [inline app/controllers/admin/courses_controller.rb]: indent

File: app/controllers/admin/courses_controller.rb [hunk 6]
```diff
@@ -0,0 +1,42 @@
+class Admin::CoursesController < AdminController
+  def index
+    @courses = Course.paginate page: params[:page], per_page: 5
+  end
+
+  def show
+    @course = Course.find params[:id]
```
Comment (MuhammadTamzid) [inline app/controllers/admin/courses_controller.rb]: no need to write this line, use load_and_authorize_resource

File: app/controllers/admin/courses_controller.rb [hunk 7]
```diff
@@ -0,0 +1,42 @@
...
+  end
+
+  def new
+    @course = Course.new
```
Comment (MuhammadTamzid) [inline app/controllers/admin/courses_controller.rb]: no need to write this line, use load_and_authorize_resource

File: app/views/admin/courses/_course_subject_fields.html.erb [hunk 8]
```diff
@@ -0,0 +1,13 @@
+<div class="form-group fields">
+  <%= f.hidden_field :id %>
+  <%= f.hidden_field :course_id %>
+  <%= f.label :subject_id, class: "col-sm-4 control-label" %>
+  <div class="col-sm-4">
+    <%= f.select :subject_id, @subjects.all.collect {|s| [s.title, s.id]},
```
Comment (dieunb) [inline app/views/admin/courses/_course_subject_fields.html.erb]:
```
@subjects.collect{|subject| [subject.title, subject.id]}
```

Don't use short word

Comment (codegracie) [inline app/views/admin/courses/_course_subject_fields.html.erb]: noted. thanks for reminding.

**Pull request merged**